cli/src/test/createApp.ts

```diff
@@ -0,0 +1,26 @@
+import { Application } from '@binaris/spice-node-client/interfaces';
+
+export function createApp( locked: boolean, lockReason?: string) {
```

Comment:
1. Inconsistent spaces inside the parentheses.

2. If this function helps test, can we use it in all the tests?  Otherwise, having a "`createApp`" that is suitable only for testing locking is inappropriate.  E.g. you might pass a `defaults: Partial<Application>` parameter, and then flatten *that* inside `makeApp`.